editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/restconfiguration/actions/AddRestElementAction.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.fusesource.ide.camel.editor.restconfiguration.actions;
+
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.jface.resource.ImageRegistry;
+import org.fusesource.ide.camel.editor.internal.UIMessages;
+import org.fusesource.ide.camel.editor.restconfiguration.RestConfigConstants;
+import org.fusesource.ide.camel.editor.restconfiguration.RestConfigEditor;
+
+/**
+ * @author lheinema
+ */
+public class AddRestElementAction extends RestEditorAction {
+	
+	/**
+	 * 
+	 * @param imageReg
```

Comment:
the javadoc is incorrect (a single parameter although there are two) and provides no information.
Either remove it or fix it with useful information